### ISSUE TITLE
c-typecyan does not meet accessibility requirements.

### ISSUE THREAD

**Issue by Charlotteis** [uswitch/ustyle#233]
I can go around fixing all the uses of it, but our `c-typecyan - #008fe9` colour does not pass accessibility requirements against a white background in any case apart from large text (WCAG AA).

What's the scope to remove it / ask people not use it on white backgrounds?

![image](https://cloud.githubusercontent.com/assets/542140/23473603/b8f35e1a-fea8-11e6-8005-279123cac916.png)

**Comment by annez** [comment]
I think we need to approach this from a `what's the scope to tweak the shade` rather then a removal or a set of rules for usage. In this scenario if we were to set rules, the amount of work to move towards adhering to that is much more complicated than triggering a **ustyle upgrade** notice.

We were aware of this when we picked the colour but it was close enough to "accessible" we decided to run with it. In the current state we'd have to revisit the palette - `c-typecyan` is used quite a bit.

Maybe the right step in this scenario is to outline the biggest culprits of the usage of this colour in an inaccessible manner?

**Comment by Charlotteis** [comment]
> I think we need to approach this from a what's the scope to tweak the shade

Sounds good to me. 



> Maybe the right step in this scenario is to outline the biggest culprits of the usage of this colour in an inaccessible manner?

Do you mean get a list of places where we use this and try and fix the worst / most visible / most important cases?

**Comment by annez** [comment]
> Do you mean get a list of places where we use this and try and fix the worst / most visible / most important cases?

Yes, but rather than try and fix them, we should understand their usage first. A search through GitHub for `color: $c-typecyan` should suffice to begin with. Maybe we should start with the worst culprits within ustyle.

**Comment by Charlotteis** [comment]
Good shout! I'll add that to my list of things to do.
